### PR TITLE
feature/abhinav3

### PR DESCRIPTION
**Pull request by abhinav30219**
## :recycle: Current situation & Problem
No personal page for Abhinav on the class application

## :bulb: Proposed solution
Added a personal page for Abhinav that satisfies assignment requirements

## :gear: Release Notes 
Abhinav's view

### Testing
Added UI tests for text appearance and button functionality

### Code of Conduct & Contributing Guidelines 

By submitting creating this pull request, you agree to follow our [Code of Conduct](https://github.com/CS342/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/CS342/.github/blob/main/CONTRIBUTING.md):
- [ x] I agree to follow the [Code of Conduct](https://github.com/CS342/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/CS342/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: CS342MemberViews/Sources/AbhinavAgarwal/AbhinavAgarwal.swift [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+// This source file is part of the CS342 2023 Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import InstructorViews
+import SwiftUI
+
```
Comment (PSchmiedmayer) [inline CS342MemberViews/Sources/AbhinavAgarwal/AbhinavAgarwal.swift:11]: Please add two lines between the import statements and the start of a type:
```suggestion


```

File: CS342MemberViews/Sources/AbhinavAgarwal/AbhinavAgarwal.swift [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+/// View displaying information about Abhinav Agarwal
+public struct AbhinavAgarwal: View {
+    @Binding var navigationPath: NavigationPath
+    private let timer = Timer.publish(every: 1.5, on: .main, in: .common).autoconnect()
+    @State var animationTick = true
+    public var body: some View {
```
Comment (PSchmiedmayer) [inline CS342MemberViews/Sources/AbhinavAgarwal/AbhinavAgarwal.swift]: Please add two empty lines between stored properties and computed properties in your types:
```suggestion
    @State var animationTick = true
    
    
    public var body: some View {
```

File: CS342MemberViews/Sources/AbhinavAgarwal/AbhinavAgarwal.swift [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+//
+
+import InstructorViews
+import SwiftUI
+
+/// View displaying information about Abhinav Agarwal
+public struct AbhinavAgarwal: View {
+    @Binding var navigationPath: NavigationPath
+    private let timer = Timer.publish(every: 1.5, on: .main, in: .common).autoconnect()
+    @State var animationTick = true
```
Comment (PSchmiedmayer) [inline CS342MemberViews/Sources/AbhinavAgarwal/AbhinavAgarwal.swift]: You can mark the state property as private as it is only used inside the view:
```suggestion
    @State private var animationTick = true
```

File: CS342MemberViews/Sources/AbhinavAgarwal/AbhinavAgarwal.swift [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+//
+// SPDX-License-Identifier: MIT
+//
+
+import InstructorViews
+import SwiftUI
+
+/// View displaying information about Abhinav Agarwal
+public struct AbhinavAgarwal: View {
+    @Binding var navigationPath: NavigationPath
```
Comment (PSchmiedmayer) [inline CS342MemberViews/Sources/AbhinavAgarwal/AbhinavAgarwal.swift:15]: As you are not using the `navigationPath` you can remove the property:
```suggestion
```

You will also need to adapt the initializer accordingly:
```swift
    /// - Parameter navigationPath: The navigation path used for navigation in the application.
    public init() { }
```

**Review by PSchmiedmayer — APPROVED** [review]
Nice job @abhinav30219! Cool to see the animation in your view!

I only had a few small small comments. Feel free to merge the PR once you have addressed the issue.

I have also merged the latest main state in your branch to make merging the PR easy as you have already resolved one merge conflict already.
Be sure that you `git pull` the latest state to your local version if you accept any of the suggestions here on GitHub and get the merge conflict to your local branch 🚀

**Review by abhinav30219 — COMMENTED** [review]
removing navigation path causes build errors

**Pull request merged**